src/main/resources/view/ItemListCard.fxml

```diff
@@ -9,27 +9,22 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.layout.FlowPane?>
 <HBox xmlns:fx="http://javafx.com/fxml/1" id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8">
-    <GridPane HBox.hgrow="ALWAYS">
-        <columnConstraints>
-            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150"/>
-        </columnConstraints>
-        <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
-            <padding>
-                <Insets top="5" right="5" bottom="5" left="15"/>
-            </padding>
-            <HBox spacing="5" alignment="CENTER_LEFT">
-                <Label fx:id="id" styleClass="cell_big_label">
-                    <minWidth>
-                        <!-- Ensures that the label text is never truncated -->
-                        <Region fx:constant="USE_PREF_SIZE"/>
-                    </minWidth>
-                </Label>
-                <Label fx:id="name" text="\$first" styleClass="cell_big_label"/>
-            </HBox>
-            <Label fx:id="quantityAndUnit" styleClass="cell_small_label" text="\$quantityAndUnit"/>
-            <Label fx:id="bought" styleClass="cell_small_label" text="\$bought"/>
-            <Label fx:id="expiry" styleClass="cell_small_label" text="\$expiry"/>
-            <FlowPane fx:id="tags" />
-        </VBox>
-    </GridPane>
+  <VBox alignment="CENTER_LEFT">
+    <padding>
+      <Insets top="5" right="5" bottom="5" left="15"/>
+    </padding>
+    <HBox spacing="5" alignment="CENTER_LEFT">
+      <Label fx:id="id">
+        <minWidth>
+          <!-- Ensures that the label text is never truncated -->
+          <Region fx:constant="USE_PREF_SIZE"/>
+        </minWidth>
+      </Label>
+      <Label fx:id="name" text="\$first"/>
+      <Label fx:id="quantityAndUnit" text="\$quantityAndUnit"/>
+      <Label fx:id="bought" text="\$bought"/>
+      <Label fx:id="expiry" text="\$expiry"/>
+    </HBox>
+    <FlowPane fx:id="tags" />
```

Comment:
Currently, the tags within the Flowpane are white on a white background. Might want to change the styling. Perhaps adding a some space between the tags will look nicer (but that's subjective so we can leave that out for now)

Reply:
Yup sure, we leave this for a future PR when it's not the MVP.